src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -119,7 +119,10 @@ private void GetDocuments(DocumentsOperationContext context, Transformer transfo
             }
             else if (HttpContext.Request.Query.ContainsKey("startsWith"))
             {
-                isLoadStartingWith = true;
+                // W.A. to distinguish between Lazily.LoadStartingWith of old client and LoadStartingWith of v4.0 new client
+                if (HttpContext.Request.Query.ContainsKey("ComingFromMultiGet") == false)
```

Comment:
When requestring Lazily.LoadStartingWith in the old client, it's being sent as part of a multi-get (Post) operation, then the multi-get handler redirects it to the normal document handler (Get) and the request looks exactly (!) like a request from the new client but the result is in a different structure (object vs array). I needed a way to tell between them